internal/levee.go

```diff
@@ -118,3 +123,16 @@ func getArgumentPropagator(c *config.Config, call *ssa.Call) ssa.Node {
 
 	return nil
 }
+
+func isFieldPropagator(c *ssa.Call, fieldPropagators map[types.Object]bool) bool {
```

Comment:
WDYT about placing this logic into the `fieldpropagator` result?
I am looking for a better encapsulation of the logic of identifying propagators.

Reply:
100% agree.